src/Microsoft.Framework.PackageManager/Building/BuildManager.cs

```diff
@@ -208,7 +208,11 @@ public bool Build()
 
             sw.Stop();
 
-            projectDiagnostics.ForEach(d => LogWarning(d.FormattedMessage));
+            if (projectDiagnostics.Any())
+            {
+                LogWarning("");
```

Comment:
Add one empty line before the file warning message to separate them from compilation warnings/errors.